public/stylesheets/account.css

```diff
@@ -105,3 +117,67 @@ main {
   margin-top: -3px;
   font-size: 9px;
 }
+
+@media (max-width: 1024px) {
+  body {
+    font-size: 10px;
+  }
+  .profile-img {
+    width: 95px;
+    height: 95px;
+  }
+  .sidebar-tab {
+    font-size: 12px;
+    padding: 10px 10px;
+  }
+  .account-first-row {
+    min-height: 131px;
+  }
+  main {
+    padding: 70px 8% 70px 5%;
+  }
+}
+
+@media (max-width: 576px) {
+  .profile-img {
+    width: 130px;
+    height: 130px;
+  }
+  .main-box {
+    display: block;
```

Comment:
I would recommend not doing it like this, it would rather be better to add rule `flex-direction: column` here.